applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -113,10 +113,10 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
             device_flag += " -DCONFIG_AUDIO_DFU=2"
         if options.min_b0n:
             device_flag += " -DCONFIG_B0N_MINIMAL=y"
-        if options.controller == Controller.sdc:
-            device_flag += " -DCONFIG_BT_LL_ACS_NRF53=n"
-            if not child_image:
-                device_flag += " -DCONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE=n"
+        if options.controller == Controller.acs_nrf53:
```

Comment:
Nice. Have you run tests with this and tried with mcuboot flags? These must be disabled for SDC.

Reply:
Yes, the code builds. When the SDC is default there is no change to the script in regard to the mcuboot flags, with the result that the code will not build if mcuboot is set to anything for a default build. These flags, including the b0n flags, can only be set if the controller is ACS_nRF53.